go/client/fuse_status_osx.go

```diff
@@ -61,3 +70,28 @@ func KeybaseFuseStatus(bundleVersion string) keybase1.FuseStatus {
 
 	return st
 }
+
+func mountInfo(fstype string) ([]keybase1.FuseMountInfo, error) {
+	out, err := exec.Command("/sbin/mount", "-t", fstype).Output()
```

Comment:
Looks like the safest thing to do is use the DiskArbitration library. I created a ticket for that here:

https://keybase.atlassian.net/browse/CORE-2122

I think in the meantime calling `/sbin/mount` is a good shortcut?